scripts/xtalk_filter_run.py

```diff
@@ -1,20 +1,18 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# Copyright 2019 the HERA Project
+# Copyright 2020 the HERA Project
 # Licensed under the MIT License
 
-"Command-line drive script for hera_cal.xtalk_filter. Only performs CLEAN Filtering"
+"""Command-line drive script for hera_cal.xtalk_filter with baseline parallelization. Only performs DAYENU Filtering"""
 
 from hera_cal import xtalk_filter
 import sys
+import hera_cal.io as io
 
-parser = xtalk_filter.xtalk_filter_argparser()
+parser = xtalk_filter.xtalk_filter_argparser(multifile=True)
```

Comment:
Didn't you remove multifile?

Reply:
yeah. I swear I went through these scripts multiple times getting rid of this. I want to say there was some kind of rebase shenanigan that happened.